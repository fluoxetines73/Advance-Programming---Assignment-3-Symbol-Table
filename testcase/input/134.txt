RPRINT
PRINT
PRINT
INSERT z number
ASSIGN z 987654 random
PRINT
INSERT f_FF number
LOOKUP z
LOOKUP f_FF
UWAAAA
ASSIGN z 123
LOOKUP f_FF
PRINT
RPRINT
INSERT x number
PRINT
UWAAAA
INSERT b number
LOOKUP x
PRINT
LOOKUP z
PRINT
RPRINT
PRINT