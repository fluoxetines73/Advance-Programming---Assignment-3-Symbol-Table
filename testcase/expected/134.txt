Invalid: ASSIGN z 987654 random